readme.md

```diff
@@ -121,6 +121,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [cgasm](https://github.com/bnagy/cgasm) - A tool that gives x86 assembly documentation. It is pronounced "SeekAzzem".
 - [cookiecutter](https://github.com/audreyr/cookiecutter) - Creates projects from cookiecutters (project templates).
 - [dockly](https://github.com/lirantal/dockly) - Interactive CLI for managing Docker containers.
+- [joker](https://github.com/amilajack/joker) - A modern and intuitive testing library for command-line apps
```

Comment:
As stated in the contribution guidelines:
"Additions should be added to the bottom of the relevant category."

Please move it to the end of the development list, i.e. below grip (line 129).